unit_tests/library/tile.cpp

```diff
@@ -212,6 +213,58 @@ template <typename algorithm> void test_tile(algorithm &osrm)
 
     BOOST_CHECK_EQUAL(number_of_turn_keys, 4);
     BOOST_CHECK(number_of_turns_found > 700);
+
+    tile_message.next();
```

Comment:
This test is getting longer and longer, I would feel better if we split this into separate sub-functions to assert on. BOOST_CHECK should work from within a function call, right?